contract/test/test-build-proposal.js

```diff
@@ -17,7 +38,9 @@ test.only('proposal builder generates compressed bundles less than 1MB', async t
 
   for (const bundle of bundles) {
     // eslint-disable-next-line @jessie.js/safe-await-separator
-    const buffer = await t.context.compressFile(bundle);
+    const buffer = await /** @type {TestContext} */ (t.context).compressFile(
```

Comment:
`t.context` type should be implied by the type of `t`. What happens without this cast?

Reply:
It's implied to be `undefined`. And says `t.context is of type unknown`

`/ava/types/test-fn.d.cts`
```js
/** The `t` value passed to test & hook implementations. */
export type ExecutionContext<Context = unknown> = {
	/** Test context, shared with hooks. */
	context: Context;

	/** Title of the test or hook. */
	readonly title: string;
	...
}
```

Reply:
That's because `test` wasn't typed. See 064eb83

Reply:
neat. thanks for this! 🚀